commit_msg.txt

```diff
@@ -0,0 +1,19 @@
+chore(STONEINTG-1189): annotate PLR when component is missing
```

Comment:
I don't think we should commit this file, but interesting to see it generated here.